wiki/en/Running-a-Server.md

```diff
@@ -36,84 +37,102 @@ However, various problems can also arise when setting up Servers - especially wh
 
 Consider using a cloud host, not your home internet connection, to get better ping times if you're having problems.
```

Comment:
```suggestion
Some internet connections don't run a Server well. A Server running on your home internet connection might struggle with a group larger than 3 or 4 participants. As an alternative, a cloud host near you can handle larger groups more reliably.
```

I've removed the claim about ping times, and instead talk about reliability and scaling. Yeah?

Reply:
Are some home connections just DOA with regard to running a Server? Maybe so...

Reply:
This "Do I need to run a Server?" section has been re-written more than almost any other part of the website. The last time we hacked at it was down to somebody saying that you'd getting better pings on the cloud. Also that we suspected people were going to this guide becuase they thought it was was the only way of ensuring an undisturbed session. BTW we shouldn't start messing with sizing statements (there's a whole separate page for that).

Tempted to say if you want to re-do this intro again, then that's for a sparate PR. This'll never get merged otherwise.

Reply:
Current line 38 is goofy.

Reply:
I'd be OK with removing the "Consider using a cloud host" bit on the ground that we shouldn't start implying that you can't host on a domestic broadband, or that there's some fixed limits to that. The line was added after a particularly vocal person demanded we point out that Jamulus would do better on dedicated hosting (which **might** be true in **most** cases).

Reply:
You can't always host on a domestic broadband, and there can be limits.

Reply:
That applies to all hosting though. We don't know anything about their domestic connectivity or hardware (we have people in Korea on synchronous gigabit speed home connections for example). So essentially saying, "Who knows whether you can do what you want on your home broadband?" isn't very helpful :-)